### DISCUSSION TITLE
Who supports Ada?

### DISCUSSION THREAD

**Discussion by mlissner** [ada-url/ada#625] [Q&A] [answered]
I swear I saw this last time I looked around here, but I can't find who is maintaining Ada, or who is behind its creation. 

I'm hoping to get Ada added to a major Python framework, so it would be helpful to know who is backing it. 

Assuming this is open information, once I have the right info, would a PR adding it to the readme be welcome?

**Comment by anonrig** [comment, marked as answer]
Yagiz Nizipli (@anonrig) and Daniel Lemire (@lemire) are behind Ada. We also have written a paper about Ada called "[Parsing millions of URLs](https://onlinelibrary.wiley.com/doi/full/10.1002/spe.3296)"

**Reply by mlissner** [reply]
I thought I saw some institutional support. Did I hallucinate that??

**Reply by lemire** [reply]
Maybe you refer to the WHATWG?

**Reply by mlissner** [reply]
No idea. Sorry for the trouble!

**Comment by lemire** [comment]
I have closed this. Users who need paid support should inquire with us.
